corehq/apps/users/views/__init__.py

```diff
@@ -463,8 +463,8 @@ def post(self, request, *args, **kwargs):
                                                                         editable_user=self.editable_user,
                                                                         is_superuser=is_superuser):
                 if current_superuser_status != is_superuser:
-                    log_user_change(self.domain, self.editable_user, changed_by_user=self.couch_user,
-                                    changed_via=USER_CHANGE_VIA_WEB,
+                    log_user_change(by_domain=self.domain, for_domain=None, couch_user=self.editable_user,
+                                    changed_by_user=self.couch_user, changed_via=USER_CHANGE_VIA_WEB,
```

Comment:
this was removed in https://github.com/dimagi/commcare-hq/pull/30302, so would not be present post update with master